printer.go

```diff
@@ -58,6 +92,36 @@ func Default() (string, error) {
 	return syscall.UTF16ToString(b), nil
 }
 
+func GetDefaultPrinterType() (string, error) {
+	printerName, _ := Default()
+	return GetPrinterType(printerName)
+}
+
+func GetPrinterType(printerName string) (string, error) {
+	b := make([]byte, 1024*10)
```

Comment:
Why 10K? How large this data is (from your own experiment)? Maybe start with that instead.

Reply:
Yes, is from my experiment, in my experiment the struct has 6k from a thermal printer with generic driver from windows